### PR TITLE
suggest-builders command

### LINKED ISSUE #144: pack users can view suggested builders
As a pack cli user
I want the option to reference builders suggested for use by the CNB project
So that I can feel confident about the builder I use when I run ```pack set-default-builder```

**Acceptance Criteria**

_Given_
I have saved the pack binary in an executable path

**Case 1:**  _Display list of suggested builders_

**When** I run ```pack suggest-builders``` 
**Then** I see the list of suggested buildpacks I would see if I tried to run pack build for the first time w/o setting a default-builder

**Case 2:**   ```pack suggest-builders``` _appears in "help" output_

**When** I run ```pack``` 
**Or** ```pack --help```
**Then** I see ```pack suggest-builders``` included in the  list of commands
**And** the associated text reads ```Display list of builders recommended by the Cloud Native Buildpacks project```

**Notes:**

* I don't think users need to be able to run --help on ```pack suggest-builders``` because the text they see in the initial help output should be sufficient for them to understand the purpose of the command

### PR DESCRIPTION
**Pull request by jromero**
Resolves #144 

Output:
image

Help:
image

Signed-off-by: Javier Romero <j.romero.1214@gmail.com>

### CODE REVIEW

File: commands/build.go [hunk 1]
```diff
@@ -90,16 +90,16 @@ func suggestSettingBuilder(logger *logging.Logger) {
 
 func suggestBuilders(logger *logging.Logger) {
 	logger.Info("Suggested builders:\n")
+
 	tw := tabwriter.NewWriter(logger.RawWriter(), 10, 10, 5, ' ', tabwriter.TabIndent)
-	for len(suggestedBuilders) > 0 {
-		n := rand.Intn(len(suggestedBuilders))
-		builders := suggestedBuilders[n]
+	for _, i := range rand.Perm(len(suggestedBuilders)) {
```
Comment (jromero) [inline commands/build.go]: This replaces the need to mutate `suggestedBuilders` and prevent consecutive tests to fail due to change of state.

File: commands/suggest_builders.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import (
+	"github.com/buildpack/pack/logging"
+	"github.com/spf13/cobra"
+)
+
+func SuggestBuilders(logger *logging.Logger) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "suggest-builders",
+		Short: "Display list of builders recommended by the Cloud Native Buildpacks project",
```
Comment (ameyer-pivotal) [inline commands/suggest_builders.go]: Maybe shorten to: `Display list of recommended builders`?

File: commands/build.go [hunk 3]
```diff
@@ -90,16 +90,16 @@ func suggestSettingBuilder(logger *logging.Logger) {
 
 func suggestBuilders(logger *logging.Logger) {
```
Comment (ameyer-pivotal) [inline commands/build.go]: While you're here, could you move `suggestBuilders` to `commands.go`? Now that it's shared, it should go with the other shared functions.

File: commands/suggest_builders.go (hunk 2 above)
Comment (jromero) [reply commands/suggest_builders.go]: I put exactly what the issue acceptance criteria states. Maybe @mgibson1121 can weigh in on this. I'm good to change it to whatever makes the most sense.

File: commands/build.go (hunk 3 above)
Comment (jromero) [reply commands/build.go]: Will do.

File: commands/suggest_builders.go (hunk 2 above)
Comment (ameyer-pivotal) [reply commands/suggest_builders.go]: Ah, my bad. Didn't realize that. I'll leave it to @mgibson1121 then :)

Comment (mgibson1121) [reply commands/suggest_builders.go]: I'm fine with @ameyer-pivotal's suggestion

**Comment by ekcasey** [comment]
Acceptance tests are fixed on master now. Travis will likely pass if you rebase.

**Pull request merged**